### DISCUSSION TITLE
How to formating value in select options ?

### DISCUSSION THREAD

**Discussion by altimusin** [filamentphp/filament#5418] [Help] [answered]
Hi All,

I trying to display select options with more than 1 field.
But the number_format is not working.

![image](https://user-images.githubusercontent.com/43429317/211185187-f000a974-bd1a-4ef9-8edd-dcf5e1a8b562.png)

```
      Select::make('catagory_id')
          ->label('Price Catagory')
          ->relationship('catagory', 'desc')
          ->getOptionLabelFromRecordUsing(
              fn (Model $record) => "{$record->desc} -> Price-1 : { number_format($record->price1,2, '.', '') }"
              )
```
Thank you

**Comment by danharrin** [comment, marked as answer]
https://www.php.net/manual/en/language.operators.string.php

you cant just run functions within strings like that
